packages/sanity/src/core/studio/components/navbar/workspace/WorkspaceMenuButton.tsx

```diff
@@ -89,6 +93,8 @@ export function WorkspaceMenuButton(props: ButtonProps) {
             })}
         </StyledMenu>
       }
+      onClose={() => props.setIsMenuOpen && props.setIsMenuOpen(false)}
```

Comment:
I recommend a slight rewrite of this code to enhance readability and to align with our ESLint rule that discourages the use of arrow functions in JSX props (`react/jsx-no-bind`).  

I suggest this update:  

```tsx
export function WorkspaceMenuButton(props: WorkspaceMenuButtonProps) {
  const {onMenuOpenChange} = props

  // ... rest of code

  const handleOnOpen = useCallback(() => {
    onMenuOpenChange?.(true)
  }, [onMenuOpenChange])

  const handleOnClose = useCallback(() => {
    onMenuOpenChange?.(false)
  }, [onMenuOpenChange])

  return (
    <MenuButton
      // ...
      onClose={handleOnClose}
      onOpen={handleOnOpen}
    />
  )
}
```

Reply:
Thanks for the input! Fixed this 😄